templates/expo-template-default/app.json

```diff
@@ -8,6 +8,7 @@
     "scheme": "myapp",
     "userInterfaceStyle": "automatic",
     "newArchEnabled": true,
+    "enableEdgeToEdge": true,
```

Comment:
sorry for catching this so late in the process but we should call this `edgeToEdgeEnabled` for consistency with `newArchEnabled`